src/BuildIt.ML/Samples/BuildIt.ML.Sample.Android/MainActivity.cs

```diff
@@ -2,21 +2,26 @@
 using Android.Content.PM;
 using Android.OS;
 using BuildIt.ML.Sample.UI;
+using Plugin.CurrentActivity;
+using Plugin.Permissions;
 
 namespace BuildIt.ML.Sample.Droid
 {
     [Activity(Label = "BuildIt.ML.Sample", Icon = "@mipmap/icon", Theme = "@style/MainTheme", MainLauncher = true, ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation)]
-    public class MainActivity : global::Xamarin.Forms.Platform.Android.FormsAppCompatActivity
+    public class MainActivity : global::Xamarin.Forms.Platform.Android.FormsApplicationActivity
```

Comment:
why this change? we should be using AppCompat where possible

Reply:
app was crashing originally, will see if it can work again upon reverting that